fastlane/lib/fastlane/actions/xcode_server_get_assets.rb

```diff
@@ -1,5 +1,6 @@
 module Fastlane
   module Actions
+    # rubocop:disable Lint/MissingKeysOnSharedArea
```

Comment:
Why are we disabling this here? Can we put a comment in here if we do really want to do this? 😊

Reply:
The issue was related to nested classes - we were checking for the `output` method in both classes. 

I improved the rubocop rule and now we are only checking for an `output` method in a class which inheritances from `Action` class 💪 

```ruby
def on_class(node)
        name, superclass, body = *node
        return unless superclass
        puts superclass.loc.name.source
        return unless superclass.loc.name.source == 'Action'

       ...
```